.travis.yml

```diff
@@ -46,47 +46,71 @@ env:
     - CCACHE_COMPRESS=1
     - CCACHE_SLOPPINESS=time_macros,file_macro
     - CCACHE_BASEDIR="$TRAVIS_BUILD_DIR"
+    # Indirection for shorter / more distinct entry in job's env listing.
+    - _TCB_=$TRAVIS_BRANCH
+
+anchors:
+  envs: &common-job-env
+    # Do not fall back to cache for "master" for PR on "release" branch.
+    _TCB_: $_TCB_
 
 jobs:
   include:
     - stage: normal builds
       os: linux
       compiler: clang-4.0
       # Use Lua so that ASAN can test our embedded Lua support. 8fec4d53d0f6
-      env: >
-        CLANG_SANITIZER=ASAN_UBSAN
-        CMAKE_FLAGS="$CMAKE_FLAGS -DPREFER_LUA=ON"
-        ASAN_SYMBOLIZE=asan_symbolize-4.0
+      env:
+        <<: *common-job-env
+        CLANG_SANITIZER: ASAN_UBSAN
+        CMAKE_FLAGS: "$CMAKE_FLAGS -DPREFER_LUA=ON"
+        ASAN_SYMBOLIZE: asan_symbolize-4.0
```

Comment:
why was `=` change to `:` ?

Reply:
Because it became a hash.  Now it's a list (to have it in the end).